sibylapp/view/utils/filtering.py

```diff
@@ -72,7 +76,32 @@ def format_func(eid):
         index=st.session_state[f"select_{eid_text}_index"],
         key=eid_text,
     )
-    pred = predictions[st.session_state[eid_text]]
+    if add_prediction:
+        pred = predictions[st.session_state[eid_text]]
+        st.sidebar.metric(context.get_term("Prediction"), config.pred_format_func(pred))
+
+
+def view_time_select(eid, row_ids, row_id_text="row_id", prefix=None, default=0):
+    def format_rowid_select(row_id):
+        return f"Timestamp: {row_id}"
```

Comment:
In a future PR we should make the "timestamp" configurable (probably by having Sibyl-API hold the real name, so no need to fix here)